### DISCUSSION TITLE
Refactoring a large markdown file into smaller pieces?

### DISCUSSION THREAD

**Discussion by wengzilla** [markdoc/markdoc#436] [General]
Is there a way to refactor a giant markdoc into smaller files that I can somehow include? Partials seems like a good option, but I have no intention of reusing the snippet across several docs...

**Comment by rpaul-stripe** [comment]
There aren't any other features built into Markdoc for content inclusion. It's worth noting that partials are implemented as a custom tag, so you could create another custom tag if you want an alternate way to do content inclusion with different behavior. Is there anything in particular that you want to be able to do that you can't do with a partial?

**Comment by wengzilla** [comment]
I think partials work for my use case! I was just thinking whether or not there might be a better solution. Thanks, @rpaul-stripe!
